cfr.json

```diff
@@ -1707,5 +1707,60 @@
       "lifetime": 3
     },
     "targeting": "firefoxVersion >= 74 && localeLanguageCode == 'en' && userPrefs.cfrFeatures && region == 'US'"
+  },
+  {
+    "id": "DOH_ROLLOUT_CONFIRMATION",
+    "targeting": "'doh-rollout.enabled'|preferenceValue && !'doh-rollout.disable-heuristics'|preferenceValue && !'doh-rollout.skipHeuristicsCheck'|preferenceValue && !'doh-rollout.doorhanger-decision'|preferenceValue && firefoxVersion >= 79",
+    "template": "cfr_doorhanger",
+    "content": {
+      "skip_address_bar_notifier": true,
+      "anchor_id": "PanelUI-menu-button",
+      "layout": "icon_and_message",
+      "text": {
+        "string_id": "cfr-doorhanger-doh-body"
+      },
+      "icon": "chrome://browser/skin/connection-secure.svg",
+      "buttons": {
+        "secondary": [
+          {
+            "label": {
+              "string_id": "cfr-doorhanger-doh-secondary-button"
```

Comment:
Just to confirm, this seems like a new string (not yet added in the remote FTL), we will have to wait for the next remote FTL publish to ship it, correct?

Reply:
It's already in central https://searchfox.org/mozilla-central/rev/a87a1c3b543475276e6d57a7a80cb02f3e42b6ed/browser/locales/en-US/browser/newtab/asrouter.ftl#272

Reply:
Yep, I saw those strings. Just wanted to confirm it :)